README.md

```diff
@@ -0,0 +1,379 @@
+
+<!-- GENERATED CODE - DO NOT ALTER THIS OR THE FOLLOWING LINES -->
+# Transform Headers
+
+[![Gravitee.io](https://img.shields.io/static/v1?label=Available%20at&message=Gravitee.io&color=1EC9D2)](https://download.gravitee.io/#graviteeio-apim/plugins/policies/gravitee-policy-transform-headers/)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/gravitee-io/gravitee-policy-transform-headers/blob/master/LICENSE.txt)
+[![Releases](https://img.shields.io/badge/semantic--release-conventional%20commits-e10079?logo=semantic-release)](https://github.com/gravitee-io/gravitee-policy-transform-headers/releases)
+[![CircleCI](https://circleci.com/gh/gravitee-io/gravitee-transform-headers.svg?style=svg)](https://circleci.com/gh/gravitee-io/gravitee-transform-headers)
+
+## Overview
+You can use the `transform-headers` policy to override headers in incoming or outgoing traffic.
+
+### Execution order
+Configured transformations headers are executed in the following order:
+
+1. Set / Replace headers
+2. Append values to existing headers (will add it if absent)
+3. Remove headers
+4. Keep only whitelisted headers
+
+### Removal
+* Headers to removed include added/appended one by this policy
+* Whitelisting applies also to added/appended headers by this policy
+
+### Native APIs
+
+Appending a list of values for a given header is not supported for Native APIs.
+
+
+
+
+
+## Phases
+The phases checked below are supported by the `transform-headers` policy:
+
+### Compatible with API type:
+
+* `PROXY`
+* `MESSAGE`
+* `NATIVE KAFKA`
+
+### Can be used in flow phases:
+
+* Publish
+* Subscribe
+* Request
+* Response
+
+## Compatibility matrix
+Strikethrough line are deprecated versions
+
+| Plugin version| APIM |
+| --- | ---  |
+|4.x|4.6.x to latest |
+|3.x|4.0.x to 4.5.x |
+|~~1.x~~|~~3.x~~ |
+
+
+## Configuration Options
+
+
+#### 
+| Name <br>`json name`  | Type <br>(constraint)  | Mandatory  | Default  | Supports <br>EL  | Supports <br>Secrets | Description  |
+|:----------------------|:-----------------------|:----------:|:---------|:----------------:|:--------------------:|:-------------|
+| Set / Replace headers<br>`addHeaders`| array|  | |  |  | Values defined here will be replaced if a header with the same name is already defined in the request.<br/>See "Set / Replace headers" section|
```

Comment:
```suggestion
| Set/replace headers<br>`addHeaders`| array|  | |  |  | Values defined here will be replaced if a header with the same name is already defined in the request.<br/>See "Set/replace headers" section.|
```

Reply:
@benoitgravitee should this mean the values defined here replace the existing values?